src/Controls/Maps/src/Pin.cs

```diff
@@ -14,6 +14,9 @@ public partial class Pin : Element
 		public static readonly BindableProperty AddressProperty = BindableProperty.Create(nameof(Address), typeof(string), typeof(Pin), default(string));
 
 		public static readonly BindableProperty LabelProperty = BindableProperty.Create(nameof(Label), typeof(string), typeof(Pin), default(string));
+
+		public static readonly BindableProperty TagProperty = BindableProperty.Create(nameof(Tag), typeof(object), typeof(Pin), default(object));
```

Comment:
Should this rather be on the base Element or VisualElement? Rather than just one, very specific element getting this - all things should get it.

https://learn.microsoft.com/en-us/uwp/api/windows.ui.xaml.frameworkelement.tagproperty?view=winrt-22621

Reply:
Ah, sorry. I mean instead of just on Pin, we maybe should add this property to _all_ elements. Like the base Element type instead. But also, I did not clarify that this was more discussion for the team to see what we are aiming for with this. 

But other than that, the code looks good - just trying to figure out _where_ it should live.

Reply:
Actually it is not that strange idea to add it to all elements because in Windows desktop apps or paid components this concept is used very widely. It would be of course better if the value was strongly typed but then it would be a different and MapPoint specific scenario because the type argument would get in the way for otherwise clean interface